### PR TITLE
chore(docs): link to demo repo

### PR DESCRIPTION
**Pull request by eshepelyuk**
This is just a _proposal_, until the whole doc wil be migrated to Gatling docs repo.

As far as my PR to demo repo was accepted, I considered that main plugin docs ( regardless where they located) 
should be reduced a little bit.

Plz suggest.

relates gatling/gatling#3953

PS: also I've added some sane exclude for GitHub worflows

### CODE REVIEW

**Review by slandelle — APPROVED** [review]

File: README.adoc [hunk 1]
```diff
@@ -40,48 +41,22 @@ details and original
...
 ====
-For those who are familiar with `Gradle` and `Gatling`, and probably having few
-already developed simulations - the recommended way is to jump right to
-<<Installation>> section and follow configuration guide to create `build.gradle`,
-organize source code and run performance tests.
+If you want to take a shortcut that skips all the configuration
+and jump right into writing and running `Gatling` simulations,
+you could check out
+https://github.com/gatling/gatling-gradle-plugin-demo[plugin demo repository]
+and use it as a starting point for creaing your `Gatling` project.
```
Comment (Isammoc) [inline README.adoc:52]:
```suggestion
and use it as a starting point for creating your `Gatling` project.
```

**Review by Isammoc — CHANGES_REQUESTED** [review]
Small typo ^^

File: README.adoc (hunk 1 above)
Comment (eshepelyuk) [reply README.adoc]: Done, thank you

**Review by Isammoc — APPROVED** [review]

**Comment by eshepelyuk** [comment]
@Isammoc @slandelle am I allowed to merge approved PRs ?

**Comment by slandelle** [comment]
@eshepelyuk Yes, you should have the permission. Please let me know if it doesn't work.

**Comment by eshepelyuk** [comment]
Well, i don't have permissions.

image

**Comment by eshepelyuk** [comment]
@Isammoc thanks for merging
but my question is still open, for the future understanding

> @Isammoc @slandelle am I allowed to merge approved PRs ?

**Comment by Isammoc** [comment]
Github permissions system is not flexible to let us tweak the rules as we want (at least one approval from a specific group)

So, only official mainteners (ie. GatlingCorp's employees) will have the right to merge into master.

Sorry for the inconvenience.

**Comment by eshepelyuk** [comment]
> Github permissions system is not flexible to let us tweak the rules as we want (at least one approval from a specific group)
> 
> So, only official mainteners (ie. GatlingCorp's employees) will have the right to merge into master.
> 
> Sorry for the inconvenience.

Well. but I had those permissions.

image

**Comment by slandelle** [comment]
That's my fault, sorry. I was trying to find an easy way that changes could only be committed on master through a PR. Not sure if that's feasible on Github atm. Permissions should be fixed now.

**Comment by eshepelyuk** [comment]
@slandelle got it thnx.

**Pull request merged**